vendor/assets/stylesheets/github_buttons.css.scss

```diff
@@ -38,76 +38,64 @@
   background-image: linear-gradient(to bottom, #fcfcfc 0, #eee 100%);
-  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#fcfcfc', endColorstr='#eeeeee', GradientType=0);
   background-repeat: no-repeat;
   border: 1px solid #d5d5d5;
 }
 .gh-btn:hover,
 .gh-btn:focus {
   text-decoration: none;
   background-color: #ddd;
-  background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0, #eee), color-stop(100%, #ddd));
-  background-image: -webkit-linear-gradient(top, #eee 0, #ddd 100%);
-  background-image: -moz-linear-gradient(top, #eee 0, #ddd 100%);
-  background-image: -ms-linear-gradient(top, #eee 0, #ddd 100%);
-  background-image: -o-linear-gradient(top, #eee 0, #ddd 100%);
+  /* stylelint-disable-next-line function-no-unknown */
+  background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0, #eee), to(#ddd));
   background-image: linear-gradient(to bottom, #eee 0, #ddd 100%);
-  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#eeeeee', endColorstr='#dddddd', GradientType=0);
   border-color: #ccc;
 }
 .gh-btn:active {
-  background-image: none;
   background-color: #dcdcdc;
+  background-image: none;
   border-color: #b5b5b5;
-  box-shadow: inset 0 2px 4px rgba(0, 0, 0, 0.15);
+  box-shadow: inset 0 2px 4px rgba(0, 0, 0, .15);
 }
 .gh-ico {
   width: 14px;
   height: 14px;
   margin-right: 4px;
-  background-image: image-url('github_icon.png');
-  background-size: 100% 100%;
-  background-repeat: no-repeat;
+  background: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='40' height='40' viewBox='12 12 40 40'%3e%3cpath fill='%23333' d='M32 13.4c-10.5 0-19 8.5-19 19 0 8.4 5.5 15.5 13 18 1 .2 1.3-.4 1.3-.9v-3.2c-5.3 1.1-6.4-2.6-6.4-2.6-.9-2.1-2.1-2.7-2.1-2.7-1.7-1.2.1-1.1.1-1.1 1.9.1 2.9 2 2.9 2 1.7 2.9 4.5 2.1 5.5 1.6.2-1.2.7-2.1 1.2-2.6-4.2-.5-8.7-2.1-8.7-9.4 0-2.1.7-3.7 2-5.1-.2-.5-.8-2.4.2-5 0 0 1.6-.5 5.2 2 1.5-.4 3.1-.7 4.8-.7 1.6 0 3.3.2 4.7.7 3.6-2.4 5.2-2 5.2-2 1 2.6.4 4.6.2 5 1.2 1.3 2 3 2 5.1 0 7.3-4.5 8.9-8.7 9.4.7.6 1.3 1.7 1.3 3.5v5.2c0 .5.4 1.1 1.3.9 7.5-2.6 13-9.7 13-18.1 0-10.5-8.5-19-19-19z'/%3e%3c/svg%3e") 0 0 / 100% 100% no-repeat;
 }
 .gh-count {
-  position: relative;
   display: none; /* hidden to start */
   margin-left: 4px;
   background-color: #fafafa;
   border: 1px solid #d4d4d4;
 }
 .gh-count:hover,
 .gh-count:focus {
-  color: #4183C4;
+  color: #0366d6;
 }
-.gh-count:before,
-.gh-count:after {
-  content: '';
+.gh-count::before,
+.gh-count::after {
   position: absolute;
   display: inline-block;
   width: 0;
   height: 0;
+  content: "";
   border-color: transparent;
   border-style: solid;
 }
-.gh-count:before {
+.gh-count::before {
   top: 50%;
   left: -3px;
-  z-index: 1;
   margin-top: -4px;
   border-width: 4px 4px 4px 0;
   border-right-color: #fafafa;
 }
-.gh-count:after {
+.gh-count::after {
   top: 50%;
   left: -4px;
-  z-index: 0;
+  z-index: -1;
```

Comment:
Is this the main change in this file?

Reply:
Nvm this comment @tnir, I finally got it :)

Reply:
Yes, this is the main change as well as a removal of unnecessary `z-index: 1`.

Reply:
Yes, I spent a lot of time to read this code, unify the code from the upstream, and understand why #1840 modified this part.